doc/classes/Particles2D.xml

```diff
@@ -305,26 +305,27 @@
 			Number of horizontal frames in [code]texture[/code].
 		</member>
 		<member name="lifetime" type="float" setter="set_lifetime" getter="get_lifetime">
-			Amount of time each particle will exist. Default value: [code]1[/code].
+			Amount of seconds each particle will exist. Default value: [code]1[/code].
```

Comment:
"Amount of seconds" doesn't sound good. In English, we usually say "amount of time" or "number of seconds" Since this can be a fractional number, I think "amount of time" sounds better. You could also say "Length of time in seconds each particle will exist"

Reply:
Isn't it important to tell what unit is used, so I'd suggest "Number of seconds" then.